### ISSUE TITLE
player forgets the map data on game load

### ISSUE THREAD

**Issue by dranitski** [CleverRaven/Cataclysm-DDA#51562]
### Describe the bug

-

### Steps To Reproduce

1. Start a new game
2. Open overworld map
3. See that player know some territory around initially
4. Save and quit
5. Load the save
6. Player know only locations he can see. For lab start - player knows nothing except one current overmap position since his overmap vision is reduced in lab

### Expected behavior

Player remembers the map on game load

### Screenshots

-

### Versions and configuration

Playing experimental version

```
commit 0144ecf5f27d8886e036bef20bf7a5637c1492d9 (grafted, HEAD -> master, origin/master, origin/HEAD)
Author: R. Stephens Summerlin <st.summerlin@gmail.com>
Date:   Sat Sep 11 13:44:07 2021 -0400
```

### Additional context

-

**Comment by dranitski** [comment]
all my lovely map notes get erased as well

**Comment by actual-nh** [comment]
This just got fixed - see #51545 for the fix, and <https://github.com/CleverRaven/Cataclysm-DDA/issues/51522#issuecomment-917535622> for a workaround for previous saves.
